src/xenia/app/emulator_window.cc

```diff
@@ -833,6 +837,9 @@ void EmulatorWindow::OnKeyDown(ui::KeyEvent& e) {
     case ui::VirtualKey::kF11: {
       ToggleFullscreen();
     } break;
+    case ui::VirtualKey::kF12: {
```

Comment:
F12 is used on debug builds to force break Xenia, so it is not a great idea to add screenshot creation on it

Reply:
Would a key like F7 or PrintScreen be okay? What key do you think would be the most fitting for a feature like this